packages/@aws-cdk/toolkit-lib/lib/api/diff/diff-formatter.ts

```diff
@@ -60,10 +77,15 @@ interface DiffFormatterProps {
    * Includes the old/current state of the stack as well as the new state.
    */
   readonly templateInfo: TemplateInfo;
+
+  /**
+   * The results of stack drift
+   */
+  readonly driftResults?: DescribeStackResourceDriftsCommandOutput;
```

Comment:
why not in template info?

Reply:
I had debated on including it in there vs outside, and my reasoning was that it wasn't as closely related to the template diff things being computed to warrant grouping them. Though going through and refactoring it now, I can see how combining them might make more sense in the grand scheme of things, and might make some of the logic in the toolkit-lib simpler as well, since it seems like that's meant to return the template info only